blocks/bricks/recurrent.py

```diff
@@ -630,3 +631,295 @@ def apply(self, *args, **kwargs):
     @apply.delegate
     def apply_delegate(self):
         return self.children[0].apply
+
+
+class RecurrentStack(BaseRecurrent, Initializable):
+    u"""Stack of recurrent networks.
+
+    Build a stack of recurrent layers from a supplied list of
+    BaseRecurrent objects. Each object must have a `sequences`,
+    `contexts`, `states` and `outputs` parameters to its `apply` method,
+    such as the ones required by the recurrent decorator from
+    :mod:`blocks.bricks.recurrent`.
+
+    The apply parameters from all layers are concatenated.
```

Comment:
"apply arguments" sounds a bit better, because the word "parameters" in Blocks already has certain semantics